packages/react/src/provider/use-open-feature-client-status.ts

```diff
@@ -1,15 +1,15 @@
 import { useEffect, useState } from 'react';
 import { useOpenFeatureClient } from './use-open-feature-client';
-import type { ProviderStatus } from '@openfeature/web-sdk';
-import { ProviderEvents } from '@openfeature/web-sdk';
```

Comment:
Is there a reason to combine these two imports? I personally like to keep types and runtime imports separate.